### PR TITLE
fix #31658 pogoda.ngs.ru

### LINKED ISSUE #31658: pogoda.ngs.ru
### Issue URL (Ads)
[https://pogoda.ngs.ru/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fpogoda.ngs.ru%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Win
AdGuard version: | 6.4.1814
Browser: | Firefox
AdGuard driver: | TDI
Filters: | AdGuard Russian,<br>AdGuard Annoyances,<br>AdGuard Base
Other extensions used: | нет

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/AdguardFilters/issues/31658

### CODE REVIEW

File: RussianFilter/sections/css_extended.txt [hunk 1]
```diff
@@ -203,6 +203,8 @@ zhzh.com.ua#?#.content > .site-name-s > h2:contains(Новини Житомир
 zhzh.com.ua#?#.aside-block:has(.adsbygoogle)
 novorosinform.org#?#.td-ss-main-sidebar .widget_text:has(> h4 > span:contains(/Новости партнеров|Актуально/))
 rgmechanics.ru#?#div:has(> center > div[id^="advertur_"])
+pogoda.ngs.ru##.article-item:has(a[href^="https://n1.ru"])
+pogoda.ngs.ru##.article-item:has(a[href*="zarplata.ru"])
```
Comment (BlazDT) [inline RussianFilter/sections/css_extended.txt]: Ext-CSS is not required here, those are the two last elements and even if they would be not the last elements, only a very small left-over would be shown.

File: RussianFilter/sections/specific.txt [hunk 2]
```diff
@@ -51,7 +51,7 @@ yogajournal.ru##.adv-frame
 bookap.info##.partner-traffic-wrapper
 hochu.ua##.moving-banners-outer
 hochu.ua##.column--right > aside > div[class="title"][style="margin-top: 15px; margin-bottom: 20px; font-size: 26px; font-family: Georgia; text-transform: uppercase;"]
-ngs24.ru##.right-column > .jtn-widget-adv
+ngs.ru,ngs24.ru##.right-column > .jtn-widget-adv
```
Comment (BlazDT) [inline RussianFilter/sections/specific.txt]: Not shown in code for me.

**Review by BlazDT — CHANGES_REQUESTED** [review]

File: RussianFilter/sections/specific.txt (hunk 2 above)
Comment (zloyden) [reply RussianFilter/sections/specific.txt]: Maybe it's regional
<details><summary>Screenshot:</summary>

Снимок экрана 2019-06-27 в 11 20 37

</details><br/>

File: RussianFilter/sections/css_extended.txt (hunk 1 above)
Comment (zloyden) [reply RussianFilter/sections/css_extended.txt]: So, what I need to do with them? If i need to block this blocks can u show me an example?

File: RussianFilter/sections/specific.txt (hunk 2 above)
Comment (Alex-302) [reply RussianFilter/sections/specific.txt]: why not just `.jtn-widget-adv`?

Comment (zloyden) [reply RussianFilter/sections/specific.txt]: I thought it can block smth usefull and need to be a more specific, but it's not.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**